quinn-proto/src/congestion.rs

```diff
@@ -0,0 +1,27 @@
+//! Logic for controlling the rate at which data is sent
+
+use std::time::Instant;
+
+mod new_reno;
+pub use new_reno::{NewReno, NewRenoConfig};
+
+/// Common interface for different congestion controllers
+pub trait Controller: Send {
+    /// Packet deliveries were confirmed
+    fn on_ack(&mut self, sent: Instant, bytes: u64, congestion_blocked: bool);
```

Comment:
Hmm, should we pre-emptively add a `now` argument here? That seems like something that might definitely be needed at least conceptually (also nice symmetry with `on_congestion_event`).

Reply:
It's needed by BBR, at least, so I suppose so.